src/utils/getConfig.ts

```diff
@@ -0,0 +1,21 @@
+import getAppDataPath from "appdata-path";
+import path from "path";
+import fs from 'fs';
+
+export function getConfig(dataPath = '') {
+  try {
+    const ad4mHostConfig = path.join(getAppDataPath(), 'ad4m-host-config.json');
```

Comment:
The file could be named like `config.ts`.
The variable `ad4m-host-config.json` can be a const value and reused in other places.